nixos/modules/services/x11/window-managers/i3.nix

```diff
@@ -23,6 +23,23 @@ in
           If left at the default value, $HOME/.i3/config will be used.
         '';
       };
+
+      debug = mkOption {
+        default = false;
+        example = true;
+        type = types.bool;
+        description = "Enable debug/verbose logging (see -V option)";
+      };
+
+      logFile = mkOption {
+        default = null;
+        example = "$HOME/.i3/i3log";
```

Comment:
Well, then you have to write smth like `${config.users.extraUsers.userName.home}/.i3/i3log`.